diff -up dialog-1.3-20170131/makefile.in.libs dialog-1.3-20170131/makefile.in
--- dialog-1.3-20170131/makefile.in.libs	2017-02-01 13:24:22.300746602 +0100
+++ dialog-1.3-20170131/makefile.in	2017-02-01 13:25:22.200905261 +0100
@@ -203,7 +203,7 @@ dialog$o \
 $(OBJECTS) : $(srcdir)/dialog.h $(srcdir)/dlg_keys.h dlg_config.h VERSION
 
 $(PROG)$x : $(LIB) dialog$o @INTLDIR_MAKE@ @INTLLIBS@
-	$(LINK) -o $@ dialog$o -L. -l@PACKAGE@ $(CFLAGS) $(LDFLAGS) $(LIBS)
+	$(LINK) -o $@ dialog$o -L. -l@PACKAGE@ $(CFLAGS) $(LDFLAGS)
 
 clean	\
 distclean \
